kyuubi-server/src/test/scala/org/apache/kyuubi/WithKyuubiServer.scala

```diff
@@ -52,13 +54,23 @@ trait WithKyuubiServer extends KyuubiFunSuite {
 
     conf.set("spark.ui.enabled", "false")
     conf.setIfMissing("spark.sql.catalogImplementation", "in-memory")
+    conf.setIfMissing("kyuubi.ha.zookeeper.connection.retry.policy", "ONE_TIME")
     conf.setIfMissing(ENGINE_CHECK_INTERVAL, 1000L)
     conf.setIfMissing(ENGINE_IDLE_TIMEOUT, 5000L)
     server = KyuubiServer.startServer(conf)
     super.beforeAll()
   }
 
   override def afterAll(): Unit = {
+    server.frontendServices.foreach {
+      case frontend: AbstractFrontendService =>
+        val sessionManager = frontend.be.sessionManager.asInstanceOf[KyuubiSessionManager]
+        sessionManager.allSessions().foreach { session =>
+          logger.warn(s"found unclosed session ${session.handle}.")
+          sessionManager.closeSession(session.handle)
```

Comment:
`closeSession` seems to throw exceptions in some cases, is it better to add a try catch?

Reply:
added.